lib/customerio/util.ex

```diff
@@ -20,6 +22,14 @@ defmodule Customerio.Util do
   """
   @type method :: :get | :post | :delete | :put | :patch
 
+  def send_behavioral_request(method, route, data_map, opts \\ []) do
+    send_request(method, @behavioral_base_route <> route, data_map, opts)
+  end
+
+  def send_api_request(method, route, data_map, opts \\ []) do
+    send_request(method, @api_base_route <> route, data_map, opts)
+  end
+
   @doc """
```

Comment:
`send_request` function seems to be called now only from inside this module. Probably, it should be marked as private?

Reply:
Agreed